backend/Tasque.WebAPI/Controllers/EntityController.cs

```diff
@@ -0,0 +1,78 @@
+﻿using AutoMapper;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Tasque.Core.BLL.Services;
+using Tasque.Core.Common.Entities.Abstract;
+
+namespace Tasque.Core.WebAPI.Controllers
+{
+    [ApiController]
```

Comment:
In my opinion this controller should have attribute [Authorize]

Reply:
Don't think so. We may allow Read operations on some entities for non-authorized users.